src/services/measure/measure.js

```diff
@@ -0,0 +1,278 @@
+   */
+  this.overlay = new ol.FeatureOverlay({
+    style: style
+  });
+};
+goog.inherits(ngeo.interaction.Measure, ol.interaction.Pointer);
+
+
+/**
+ * @inheritDoc
+ */
+ngeo.interaction.Measure.prototype.setMap = function(map) {
+  goog.base(this, 'setMap', map);
+  this.overlay.setMap(map);
+
+  map.addInteraction(this.drawInteraction);
+
+  this.drawInteraction.on('drawstart', goog.bind(this.onDrawStart_, this));
+  this.drawInteraction.on('drawend', goog.bind(this.onDrawEnd_, this));
+};
+
+
+/**
+ * Handle draw interaction `drawstart` event.
+ * @param {ol.DrawEvent} evt
+ * @private
+ */
+ngeo.interaction.Measure.prototype.onDrawStart_ = function(evt) {
+  this.sketchFeature = evt.feature;
+  this.overlay.getFeatures().clear();
+  this.createMeasureTooltip_();
+};
+
+
+/**
+ * Handle draw interaction `drawend` event.
+ * @param {ol.DrawEvent} evt
+ * @private
+ */
+ngeo.interaction.Measure.prototype.onDrawEnd_ = function(evt) {
+  goog.dom.classlist.add(this.measureTooltipElement_, 'tooltip-static');
+  this.measureTooltip_.setOffset([0, -7]);
+  this.sketchFeature = null;
+};
+
+
+/**
+ * Creates a new help tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.createHelpTooltip_ = function() {
+  this.removeHelpTooltip_();
+  this.helpTooltipElement_ = goog.dom.createDom(goog.dom.TagName.DIV);
+  goog.dom.classlist.add(this.helpTooltipElement_, 'tooltip');
+  this.helpTooltip_ = new ol.Overlay({
+    element: this.helpTooltipElement_,
+    offset: [15, 0],
+    positioning: 'center-left'
+  });
+  this.getMap().addOverlay(this.helpTooltip_);
+};
+
+
+/**
+ * Destroy the help tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.removeHelpTooltip_ = function() {
+  this.getMap().removeOverlay(this.helpTooltip_);
+  if (!goog.isNull(this.helpTooltipElement_)) {
+    this.helpTooltipElement_.parentNode.removeChild(this.helpTooltipElement_);
+  }
+  this.helpTooltipElement_ = null;
+  this.helpTooltip_ = null;
+};
+
+
+/**
+ * Creates a new measure tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.createMeasureTooltip_ = function() {
+  this.removeMeasureTooltip_();
+  this.measureTooltipElement_ = goog.dom.createDom(goog.dom.TagName.DIV);
+  goog.dom.classlist.addAll(this.measureTooltipElement_,
+      ['tooltip', 'tooltip-measure']);
+  this.measureTooltip_ = new ol.Overlay({
+    element: this.measureTooltipElement_,
+    offset: [0, -15],
+    positioning: 'bottom-center'
+  });
+  this.getMap().addOverlay(this.measureTooltip_);
+};
+
+
+/**
+ * Destroy the help tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.removeMeasureTooltip_ = function() {
+  if (!goog.isNull(this.measureTooltipElement_)) {
+    this.measureTooltipElement_.parentNode.removeChild(
+        this.measureTooltipElement_);
+    this.measureTooltipElement_ = null;
+    this.measureTooltip_ = null;
+  }
+};
+
+
+/**
+ * @inheritDoc
+ */
+ngeo.interaction.Measure.prototype.setActive = function(active) {
```

Comment:
You could also listen to change event on the `active` property instead of overloading `setActive`, just like you do for the `DrawAzimut` interaction.